src/app/state/account/saga.ts

```diff
@@ -15,6 +15,7 @@ import { getAccountBalanceWithFallback } from '../../lib/getAccountBalanceWithFa
 import { walletActions } from '../wallet'
 
 const ACCOUNT_REFETCHING_INTERVAL = process.env.REACT_APP_E2E_TEST ? 5 * 1000 : 30 * 1000
+const TRANSACTIONS_UPDATE_DELAY = 35 * 1000 // Measured between 8 and 31 second additional delay after balance updates
```

Comment:
Should there be a different delay on mainnet opposed to testnet? As the issue comment suggests, there is only 8sec delay on mainnet. Why should an user wait x3 the time, when it could have been updated faster?

Reply:
Mainnet often takes 20+ seconds

Reply:
Can we adjust that for the mainnet? 15sec is still a lot of time.